Local Pods/GitHubAPI/GitHubAPITests/Discussions/discussions.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "author": {
+            "login": "octocat",
+            "id": 1,
+            "node_id": "MDQ6VXNlcjE=",
+            "avatar_url": "https://github.com/images/error/octocat_happy.gif",
+            "gravatar_id": "",
+            "url": "https://api.github.com/users/octocat",
+            "html_url": "https://github.com/octocat",
+            "followers_url": "https://api.github.com/users/octocat/followers",
+            "following_url": "https://api.github.com/users/octocat/following{/other_user}",
+            "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
+            "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
+            "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
+            "organizations_url": "https://api.github.com/users/octocat/orgs",
+            "repos_url": "https://api.github.com/users/octocat/repos",
+            "events_url": "https://api.github.com/users/octocat/events{/privacy}",
+            "received_events_url": "https://api.github.com/users/octocat/received_events",
+            "type": "User",
+            "site_admin": false
+        },
+        "body": "Hi! This is an area for us to collaborate as a team.",
+        "body_html": "<p>Hi! This is an area for us to collaborate as a team</p>",
+        "body_version": "0d495416a700fb06133c612575d92bfb",
+        "comments_count": 0,
+        "comments_url": "https://api.github.com/teams/2343027/discussions/1/comments",
+        "created_at": "2018-01-25T18:56:31Z",
+        "last_edited_at": null,
+        "html_url": "https://github.com/orgs/github/teams/justice-league/discussions/1",
+        "node_id": "MDE0OlRlYW1EaXNjdXNzaW9uMQ==",
+        "number": 1,
+        "pinned": false,
+        "private": false,
+        "team_url": "https://api.github.com/teams/2343027",
+        "title": "Our first team post",
+        "updated_at": "2018-01-25T18:56:31Z",
+        "url": "https://api.github.com/teams/2343027/discussions/1",
+        "reactions": {
```

Comment:
This is in the [example json](https://developer.github.com/v3/teams/discussions/#response) but we shouldn't try to decode it yet; it needs an [Accept header](https://developer.github.com/v3/teams/discussions/#list-discussions) (`application/vnd.github.squirrel-girl-preview`)... but we already [have one])https://github.com/GitHawkApp/GitHawk/pull/1987/files#diff-c452e40c33cddad76f5cc2ba892c06d8R22).

Should we refactor that to accept a `[(String, String)]` instead of a `Dictionary`?